frontend/fingertips-frontend/mock/server/handlers.ts

```diff
@@ -15,10 +19,108 @@ export const handlers = [
   http.get(`${baseURL}/WeatherForecast`, async () => {
     const resultArray = [[getGetWeatherForecast200Response(), { status: 200 }]];
 
+    return HttpResponse.json(...resultArray[next() % resultArray.length]);
+  }),
+  http.get(`${baseURL}/indicators`, async () => {
+    const resultArray = [
+      [getFilterIndicators200Response(), { status: 200 }],
+      [getFilterIndicators400Response(), { status: 400 }],
+      [getFilterIndicators500Response(), { status: 500 }],
+    ];
```

Comment:
Isn't this just going to cycle through giving error responses 2/3 of the time? I think that might just make things confusing when using the mocks.
```suggestion
    const resultArray = [
      [getFilterIndicators200Response(), { status: 200 }]
    ];
```